src/web/src/average.ecpp

```diff
@@ -136,7 +137,7 @@ bool database_ready;
             // possible compinations:
             //      ( relative != "" ) XOR ( start_ts != "" AND end_ts != "" )
             if (csv != "yes") { // for non-csv we parse "relative", for csv by BIOS-3225 we do not yet
-                http_die ("request-param-required", "start_ts or relative");
+                http_die ("request-param-required", zsys_sprintf ((char *) "%s %s %s", "start_ts", TRANSLATE_ME ("or"), "relative"));
```

Comment:
I'd rather use "%s or %s" as start_ts and relative are variables, or is a string content, and "%s or %s" would make more sense to Translator

Reply:
translating start_ts and 'relative' does not make sence

Reply:
Yes, but translating simply "or" sounds weird to me, without context:
"or" or "$variable1 or $variable2", which sounds better to you if you'd have to translate to Czech?